hoth_api.yaml

```diff
@@ -402,9 +402,44 @@ components:
         templatePayload:
           type: object
           oneOf:
+            - $ref: "#/components/schemas/EvalPlan"
             - $ref: "#/components/schemas/DescriptionOfWork"
             - $ref: "#/components/schemas/IndependentGovernmentCostEstimate"
-    IndependentGovernmentCostEstimate:
+   EvalPlan:
+     description: Provides the basis of evaluation for an Aquisition Package
+     type: object
+     properties:
+        source_selection:
+          type: string
+          enum:
+            - NO_TECH_PROPOSAL
+            - TECH_PROPOSAL
+            - SET_LUMP_SUM
+            - EQUAL_SET_LUMP_SUM
+        method:
+          type: string
+          enum:
+            - LPTA
+            - BVTO
+            - BEST_USE
+            - LOWEST_RISK
+        standard_specifications:
+          type: array
+          items:
+            type: string
+        custom_specifications:
+          type: array
+          items:
+            type: string
+        standard_differentiators:
+          type: array
+          items:
+            type: string
+        custom_differentiators:
+          type: array
+          items:
+            type: string
+   IndependentGovernmentCostEstimate:
```

Comment:
These lines need to shift by a space

Easier to see in this view

![image](https://user-images.githubusercontent.com/77642966/203618942-98dd85f5-00e7-4f40-9f1c-f04862ca535a.png)


```suggestion
    EvalPlan:
      description: Provides the basis of evaluation for an Aquisition Package
      type: object
      properties:
         source_selection:
           type: string
           enum:
             - NO_TECH_PROPOSAL
             - TECH_PROPOSAL
             - SET_LUMP_SUM
             - EQUAL_SET_LUMP_SUM
         method:
           type: string
           enum:
             - LPTA
             - BVTO
             - BEST_USE
             - LOWEST_RISK
         standard_specifications:
           type: array
           items:
             type: string
         custom_specifications:
           type: array
           items:
             type: string
         standard_differentiators:
           type: array
           items:
             type: string
         custom_differentiators:
           type: array
           items:
             type: string
    IndependentGovernmentCostEstimate:
```

Reply:
Fixed in https://github.com/dod-ccpo/atat-web-api/pull/1363/commits/df26bbff2c3a0cf385d523e02ec5a19948007366